src/com/sun/ts/tests/signaturetest/SigTestEE.java

```diff
@@ -299,7 +299,7 @@ public void signatureTest() throws Fault {
   /**
    * Called by the test framework to run this test.  This method utilizes the
    * state information set in the setup method to run.  This test validates
-   * that the jakarta.transaction.xa type is not in the JTA API jar
```

Comment:
We are going to regenerate/update the signaturetest folder changes separately, so this changes in signaturetest aren't needed.

Reply:
Reverted all .java in `src/com/sun/ts/tests/signaturetest` back to master versions.